docs/overview/office-add-in-availability.md

```diff
@@ -78,13 +106,6 @@ To work as expected, your Office Add-in might depend on a specific Office host,
         - Content<br>
         - <a href="https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/add-in-commands-requirement-sets">Add-in Commands</a></td>
```

Comment:
Michael thinks that Add-in Commands (ribbon extensibility) is also not in 2016. Yi should know for sure.